### ISSUE TITLE
Opsworks error undefined method 'source_url'

### ISSUE THREAD

**Issue by dumityty** [parallels-cookbooks/cookbook-logstash-forwarder#3]
I was trying to set up this cookbook on an AWS Opsworks environment but it failed with the following error:

```
ERROR: Could not read /opt/aws/opsworks/current/merged-cookbooks/logstash-forwarder into a Chef object: undefined method `source_url'
```

(currently Opsworks running off Chef 11.10.4, Berkshelf 3.2.0)

I just wanted to make you aware of this and it might help other people who might bump into this issue.
I don't know what could be done about it at the moment. 

It looks like other cookbooks are having the same issue:
https://github.com/opscode-cookbooks/aws/issues/122

**Comment by legal90** [comment]
It is related to https://github.com/chef/chef/issues/2937, e.q. `source_url` in `metadata.rb` is supported only for Chef 12 and higher 

@Kasen What do you think about removing this directive to make this cookbook compatible with Chef 11?

**Comment by Kasen** [comment]
I think it is good idea. I do it ASAP.

**Comment by Kasen** [comment]
Fixed in version 0.2.0
